source/opt/basic_block.cpp

```diff
@@ -212,8 +212,11 @@ BasicBlock* BasicBlock::SplitBasicBlock(IRContext* context, uint32_t label_id,
                                         iterator iter) {
   assert(!insts_.empty());
 
-  BasicBlock* new_block = new BasicBlock(MakeUnique<Instruction>(
-      context, SpvOpLabel, 0, label_id, std::initializer_list<Operand>{}));
+  std::unique_ptr<BasicBlock> new_block_temp =
+      MakeUnique<BasicBlock>(MakeUnique<Instruction>(
+          context, SpvOpLabel, 0, label_id, std::initializer_list<Operand>{}));
+  BasicBlock* new_block = new_block_temp.get();
+  function_->InsertBasicBlockAfter(std::move(new_block_temp), this);
```

Comment:
This code is now inserting the new block to the function.  Is the change in semantics needed?  I don't recall whether there was a good reason to do the previous semantics.  I think this is simpler, but this may need changes in every caller.

Reply:
Yes, I already changed all of the calls to this function.  I forgot to update the function comment, so I did that.